easyvalidation-core/src/main/java/com/wajahatkarim3/easyvalidation/core/view_ktx/EditTextKtx.kt

```diff
@@ -7,424 +7,359 @@ import java.math.BigInteger
 import java.util.regex.Pattern
 import kotlin.math.max
 
-fun EditText.validator() : Validator
-{
+fun EditText.validator(): Validator {
     return Validator(text.toString())
 }
 
-fun EditText.nonEmpty() : Boolean
-{
-    return validator().nonEmpty().check()
+fun EditText.nonEmpty(errorMsg: String? = null): Boolean {
+    return validator().nonEmpty(errorMsg).check()
 }
 
-fun EditText.nonEmpty(callback: (message: String) -> Unit) : Boolean
-{
-    return validator().nonEmpty()
+fun EditText.nonEmpty(callback: (message: String) -> Unit, errorMsg: String? = null): Boolean {
+    return validator().nonEmpty(errorMsg)
             .addErrorCallback {
                 callback.invoke(it)
             }
             .check()
 }
 
-fun EditText.minLength(minLength: Int) : Boolean
-{
+fun EditText.minLength(minLength: Int): Boolean {
```

Comment:
The `errorMsg` variable should be added in all the validation methods, not only in `nonEmpty()` checks. For example, for `minLength()` here, this should be done like this:
```kotlin
fun EditText.minLength(minLength: Int, errorMsg: String? = null): Boolean {
       return validator()
       .minLength(minLength, errorMsg)              // Pass errorMsg here
       .check()
}
```
And similarly this should be done for other method with ```callback``` parameter also.